src/tests/nativeaot/SmokeTests/SharedLibrary/Library.cs

```diff
@@ -0,0 +1,176 @@
+// Generated by `wit-bindgen` 0.32.0. DO NOT EDIT!
+// <auto-generated />
```

Comment:
Could you trim this to the most minimum set [of exports] we need to satisfy in order for the test to build?

I think for this test it would be useful to have the lowest-level view of the system, i. e. do without the auto-generated code.

Reply:
I could write the same code by hand, but it would look pretty much the same, minus the `None` and `Result` definitions (which we should teach `wit-bindgen` not to emit when they're not needed).  I guess I'm not seeing an advantage to doing that given that the generated code is pretty readable already and doesn't hide any of the details.  And if/when we add additional entrypoints in the future, it will be convenient to regenerate the code from the WIT.

Happy to do whatever is needed to get this PR merged, but I don't quite understand the motivation for this suggestion.  As a compromise, perhaps I could update `wit-bindgen` to omit unneeded declarations and regenerate these bindings with that change?

Reply:
> but I don't quite understand the motivation for this suggestion

The main motivation is simplifying this test. Take the current call chain for `ReturnsPrimitiveInt`, for example. Right now it is:
```
LibraryWorldImpl::ReturnsPrimitiveInt [implementation]
[There is a static abstract interface ILibraryWorld that I don't think has any significant purpose]
LibraryWorld::wasmExportReturnsPrimitiveInt [Actual UCO export]
```
These are "not strictly necessary" steps and types.

The more broader point, though, is that I would consider "handrolling" here a feature, not a bug (including when adding new entrypoints).

Reply:
Yeah, I can see how a "only what's strictly necessary" approach can simplify debugging; I don't necessarily agree we need to take it that far, but I'm not going to be stubborn about it.  I'll eliminate the generated code.